index.html

```diff
@@ -708,6 +708,9 @@ <h3 class="col-md-11 col-lg-11">Electrify</h3>
   <!-- end footer -->
   <script src="./scripts/mobile-pillars.js"></script>
   <script src="https://code.jquery.com/jquery-3.1.1.js" integrity="sha256-16cdPddA6VdVInumRGo6IbivbERE8p7CQR3HzTBuELA=" crossorigin="anonymous"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
```

Comment:
@ananavati the only use we have to for the bootstrap.js that I'm aware of is the carousel. We put all the scripts at the bottom of the page, for optimized rendering of html and css content. We only load bootstrap.js once.